blocks/blogs/blogs.css

```diff
@@ -0,0 +1,228 @@
+    border-radius: var(--space-8px);
+    padding: var(--space-15px);
+    height: 100%;
+}
+
+.block.blogs .title {
+    border-bottom: var(--space-1px) solid var(--border-color-dark-grey);
+    text-align: center;
+    vertical-align: baseline;
+    margin-bottom: 1.25rem;
+    padding-bottom: var(--space-10px);
+
+}
+
+.block.blogs .title h2 {
+    font-size: var(--h2-font-size-mobile);
+    line-height: var(--line-height-normal);
+    color: var(--primary);
+    text-transform: uppercase;
+    font-weight: var(--font-weight-bold);
+    margin-bottom: 0;
+}
+
+.block.blogs .carousel-container {
+    background-color: var(--background-color-light-grey);
+    border: 0.06rem solid rgb(14 31 53 / 20%);
+    border-bottom-left-radius: 0.9rem;
+    border-bottom-right-radius: 0.9rem;
+}
+
+.block.blogs .carousel-slider .carousel-track,
+.block.blogs .carousel-slider .carousel-list {
+    transform: translate3d(0, 0, 0);
+}
+
+.block.blogs .carousel-slider .carousel-list {
+    overflow: hidden;
+}
+
+.block.blogs .carousel-slider .carousel-track {
+    position: relative;
+    left: 0;
+    top: 0;
+    display: flex;
+    transition: transform 0.5s ease;
+    min-height: 17.5rem;
+}
+
+.block.blogs .carousel-slider .carousel-track > :nth-child(-n+1){
+    flex:1;
+    display: block;
+}
+
+.block.blogs .carousel-slider .carousel-track > :nth-child(n+2){
+    display:none;
+}
+
+
+
+
+
+.block.blogs .carousel-slider .carousel-card .article {
+    padding: var(--space-8px) 0;
+    width: 100%;
+}
+
+
+
+.block.blogs .carousel-card .picture-wrapper {
+    background: #314467;
+    display: block;
+    border-radius: var(--space-10px);
+}
+
+.block.blogs .carousel-card .picture-wrapper  picture {
+    min-height: 10rem;
+    display: block;
+    position: relative;
+    overflow: hidden;
+}
+
+.block.blogs .carousel-card .picture-wrapper  picture img {
+    object-fit: initial;
+    opacity: 0.8;
+    width: 100%;
+    height: 100%;
+    transition: 0.3s;
+    position: absolute;
+
+}
+
+.block.blogs .carousel-card .article-content a {
+    font-size: var(--heading-font-size-ss);
+    line-height: var(--line-height-medium);
+    color: var(--color-light-black);
+    height: 2.625rem;
+    font-weight: var(--font-weight-semibold);
+    display: -webkit-box;
+    word-break: break-word;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    -webkit-line-clamp: 2;
+    -webkit-box-orient: vertical;
+    margin-bottom: var(--space-12px);
+}
+
+
+
+.block.blogs .carousel-card .article-content .article-text {
+    font-size: var(--body-font-size-s);
+    line-height: var(--line-height-small);
+    color: var(--color-light-black);
+    height: 3.375rem;
+    margin-bottom: 1.25rem;
+    margin-top: var(--space-5px);
+    word-break: break-word;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    display: -webkit-box;
+    -webkit-line-clamp: 3;
+    -webkit-box-orient: vertical;
```

Comment:
We should avoid vendor specific css attributes unless absolutely needed.

Reply:
As discussed in Sync up today, to achieve the same effect as original site. We have to keep use these css